icu4c/source/test/perf/convperf/convperf.cpp

```diff
@@ -144,7 +144,7 @@ UPerfFunction* ConverterPerformanceTest::TestICU_OpenAllConverters() {
 
 UPerfFunction* ConverterPerformanceTest::TestICU_UTF8_FromUnicode(){
     UErrorCode status = U_ZERO_ERROR;
-    ICUFromUnicodePerfFunction* pf = new ICUFromUnicodePerfFunction("utf-8",utf8_uniSource, UPRV_LENGTHOF(utf8_uniSource), status);
+    ICUFromUnicodePerfFunction* pf = new ICUFromUnicodePerfFunction("utf-8", toUCharPtr(UnicodeString(utf8_uniSource).getBuffer()), UPRV_LENGTHOF(utf8_uniSource), status);
```

Comment:
`utf8_uniSource` is defined as `WCHAR` and `ICUFromUnicodePerfFunction` takes a UChar.
Why do we need to create a temporary UnicodeString here?
Can this just be a simple cast instead? Or `toOldUCharPtr`?

Reply:
Yes these should just be a simple cast, for some reason I thought I needed these extra steps when reading the ICU 59 changes. Thanks again Jeff!